src/components/NatureOfBusiness/NatureOfBusiness.vue

```diff
@@ -90,20 +87,22 @@
 import { Component, Emit, Prop, Vue, Watch } from 'vue-property-decorator'
 import NaicsHelpText from './NaicsHelpText.vue'
 import NaicsResult from './NaicsResult.vue'
-import { EmptyNaics, NaicsIF, NaicsResultIF } from '@bcrs-shared-components/interfaces'
+import { NaicsIF, NaicsResultIF } from '@bcrs-shared-components/interfaces'
 
 enum States {
-  INITIAL,
-  SEARCHING,
-  SHOW_RESULTS,
-  NO_RESULTS,
-  SUMMARY
+  INITIAL = 'initial',
+  SEARCHING = 'searching',
+  SHOW_RESULTS = 'show results',
+  NO_RESULTS = 'no results',
+  SUMMARY = 'summary'
```

Comment:
With values not specified, the keys are automatically numbered from 0.

Declaring values like this makes it easier to understand during debugging. For example, the following displays a string instead of a number :)
```
<pre>state={{state}}</pre>
```